ELLog/Destinations/LogConsoleDestination.swift

```diff
@@ -19,7 +19,7 @@ The default behavior is:
     showTimestamp = false
 */
 @objc(ELLogConsoleDestination)
-public class LogConsoleDestination: LogDestinationBase {
+open class LogConsoleDestination: LogDestinationBase {
```

Comment:
This is correct for matching the current behavior but I'm wondering if it would now make more sense to make this concrete. There is already an abstract base type, `LogDestinationBase`, that can be subclassed for customization so I think it would be better to make `LogConsoleDestination` a concrete type by removing `open`.